### PR TITLE
Add LocalTap from Etsy, used to side-step lack of SequenceFile implementation for Cascading local mode

### PR DESCRIPTION
**Pull request by mrwalker**
I wasn't clear exactly where to locate this in source, so I opted to just keep it in the package have it in at Etsy.  Feel free to move.

### CODE REVIEW

File: src/jvm/com/etsy/cascding/tap/local/LocalTap.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        this.path = path;
+
+        /*
+         * LocalTap requires your system Hadoop configuration for defaults to
+         * supply the wrapped Lfs. Make sure you have your serializations and
+         * serialization tokens defined there.
+         */
+        defaults = new JobConf();
+
+        // HACK: c.t.h.TextLine checks this property for .zip files; the check
```
Comment (sritchie) [inline src/jvm/com/etsy/cascding/tap/local/LocalTap.java:68]: Hack... is this some sort of Etsy practice?

**Comment by sritchie** [comment]
Hey, this is awesome. Can I get you to resubmit this pull req against develop? I've made develop the default for future reqs.

File: src/jvm/com/etsy/cascding/tap/local/LocalTap.java (hunk 1 above)
Comment (mrwalker) [inline src/jvm/com/etsy/cascding/tap/local/LocalTap.java:68]: Heh, not that I'm aware of.  Just calling attention to what feels like a dirty hack.  Also, this was written for hack week.

**Comment by mrwalker** [comment]
Will do.

Also, I'll review this a bit more with respect to properties -- cwensel just merged a bit of work I did in response to issues with tap properties in local mode.  I most likely won't go so far as to learn Clojure to contribute tests, though =)

File: src/jvm/com/etsy/cascding/tap/local/LocalTap.java (hunk 1 above)
Comment (sritchie) [inline src/jvm/com/etsy/cascding/tap/local/LocalTap.java:68]: Just messing around :)

**Pull request closed without merging**